src/logger/loggerConfig.c

```diff
@@ -690,14 +697,24 @@ int flashLoggerConfig(void)
                                sizeof (LoggerConfig));
 }
 
+
+static bool _config_size_changed(void) {
+    bool changed = false;
+
+    if (g_savedLoggerConfig.config_size != sizeof(LoggerConfig)) {
+        changed = true;
+        pr_warning(_LOG_PFX "size of LoggerConfig changed\r\n");
+    }
+    return changed;
+}
+
 static bool checkFlashDefaultConfig(void)
 {
         const VersionInfo sv = g_savedLoggerConfig.RcpVersionInfo;
-        bool changed = version_check_changed(&sv, "Config DB");
+        bool changed = version_check_changed(&sv) || _config_size_changed();
```

Comment:
This helps prevent RC from crashing when we muck with the size of the logger config. It's an extra layer of protection on top of checking the major / minor version #